ci/bazelrc_github

```diff
@@ -0,0 +1,34 @@
+# Limit resources since GitHub Actions VMs have 2 cores and 7 GB RAM.
```

Comment:
Slightly nicer if you don't mind I think to instead fork `ci/bazelrc_travis` from our current file, and then edit it in place for GitHub Actions.  This way 1) we see the diff and 2) we associate the history with the file we intend to keep long-term.

Reply:
Ah, good idea. Thanks.